modules/promotion/src/Annotation/PromotionCondition.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\commerce_promotion\Annotation;
+
+use Drupal\Core\Condition\Annotation\Condition;
+
+/**
+ * Defines the promotion condition plugin annotation object.
+ *
+ * Plugin namespace: Plugin\Commerce\PromotionCondition.
+ *
+ * @Annotation
+ */
+class PromotionCondition extends Condition {
```

Comment:
Commerce annotations start with Commerce. So this should be CommercePromotionCondition. If the same problem exists with offers we can merge and open a followup.